examples/Popover.focus.tsx

```diff
@@ -0,0 +1,54 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as React from 'react';
+import { Button, Flex, Popover, Surface, Text } from '@itwin/itwinui-react';
+
+export default () => {
+  const idPrefix = React.useId();
+  const headingId = `${idPrefix}-label`;
+  const descriptionId = `${idPrefix}-description`;
+
+  const [isOpen, setIsOpen] = React.useState(false);
+
+  return (
+    <Popover
+      applyBackground
+      aria-labelledby={headingId}
+      aria-describedby={descriptionId}
+      visible={isOpen}
+      onVisibleChange={setIsOpen}
+      style={{ maxWidth: '45ch', border: 'none' }}
+      content={
+        <Surface elevation={0}>
```

Comment:
I had to add `border: none` override because Surface always insists on bringing its own border.

Also, for some reason `Surface.Body` only brings horizontal padding not vertical.
![](https://github.com/iTwin/iTwinUI/assets/9084735/343755f0-fbd7-452b-843d-a1fe5801d45f)

@FlyersPh9 Is this intentional?

Reply:
`data-iui-padded="true"` does have vertical padding, it's just very little.
![Screenshot 2023-09-28 at 4 28 04 PM](https://github.com/iTwin/iTwinUI/assets/849817/8f334e50-519b-4425-8f06-0586469fdf17)
The examples on [the surface demo page](https://itwin.github.io/iTwinUI/css/surface.html) look nice, but the agree button in your screenshot does not look so great.  Was the padding changed when we implemented `scrollbar-gutter`?

Btw, just now found out [`scrollbar-gutter` doesn't work in Safari](https://caniuse.com/mdn-css_properties_scrollbar-gutter). 😓